src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -765,6 +765,45 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('auto_mapping')
```

Comment:
automap? or automapping?

Reply:
Doctrine Bundle already uses `auto_mapping` (with the underscore): https://symfony.com/doc/current/reference/configuration/doctrine.html#doctrine-orm-configuration

I suggest to keep the same name for consistency.